resources/assets/uad_lists.json

```
@@ -7202,11 +7202,11 @@
   },
   "com.android.carrierconfig": {
     "list": "Aosp",
-    "description": "Dynamically provides configuration for the carrier network.\nThe config contains: Roaming networks, Voicemail settings, SMS/MMS settings, VoLTE/IMS settings, and more.\nIf a carrier app is installed it will be queried for overrides to these settings.\nSeems to run on boot and when you swap SIM?\nhttps://source.android.com/devices/tech/config/carrier\nhttps://cs.android.com/android/platform/superproject/+/master:packages/apps/CarrierConfig/src/com/android/carrierconfig/DefaultCarrierConfigService.java",
+    "description": "Platform carrier config app, bundled with system image.\nMay fail at configuring a new carrier/SIM.\nhttps://cs.android.com/android/platform/superproject/main/+/main:packages/services/Telephony/src/com/android/phone/CarrierConfigLoader.java",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Advanced"
+    "removal": "Recommended"
```

```suggestion
    "removal": "Expert"
```
Went the wrong way on this one.